components/fxa-client/src/commands/send_tab.rs

```diff
@@ -40,16 +42,31 @@ impl EncryptedSendTabPayload {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct SendTabPayload {
     pub entries: Vec<TabHistoryEntry>,
+    // XXX - tests don't fail yet even though none add these 2 new items and
+    // they aren't marked as optional for serde, which seems odd!
```

Comment:
I marked these as optional in case we're dealing with an old Firefox that sends the `streamID` unencrypted, or a tab coming from pre-79 Firefox—so we'll still show the tab, even if we can't record telemetry for it. WDYT?